internal/kafka/internal/workers/clusters_mgr.go

```diff
@@ -1225,6 +1225,14 @@ func (c *ClusterManager) setKafkaPerClusterCountMetrics() error {
 		return err
 	} else {
```

Comment:
Wdyt of moving the `err != nil` check? This way we won't need to surround the rest of the code with an `else`.

Reply:
I've updated the logic.
Can you have another look?

> +1 on revisiting, both metrics result in a streaming unit count so they're the exact same.

I'll create a JIRA for this.

Reply:
of course, I  updated the change and forgot to push. Everything has been pushed now.